test/Sentry.Unity.Tests/UnityEventProcessorTests.cs

```diff
@@ -176,42 +176,6 @@ static SentryEvent CreateSentryEvent()
                     }
                 };
         }
-
-        [Test]
-        public void SentrySdkCaptureEvent_WrongDsn_CorrectException()
```

Comment:
This test isn't really doing anything other than testing DNS and our internal logging. Also, it started failing because of a change in the .NET SDK.